lib/azuremetadata/azuremetadatautils.py

```diff
@@ -0,0 +1,95 @@
+from azuremetadata import azuremetadata
+
+
+class QueryException(Exception):
+    pass
```

Comment:
What's the purpose of the exception class with no exception methods that gives it a scope ? Do you plan to extend this while the code evolves ? I'm asking because other code that will change was commented with a FIXME block. This one stands alone and I was wondering about it.

Reply:
All of those exceptions are being caught here:
https://github.com/SUSE-Enceladus/azuremetadata/blob/4d330dd020d2dd8779929c5c8c2d1c1a8ba39c80/azuremetadata#L30-L36

Catching the generic `Exception` is frowned upon pretty much in all languages ("Gotta catch them all"), hence the subclass.